doc/source/man.rst

```diff
@@ -32,206 +32,220 @@ rho performs four major tasks:
 
-The ``rho auth list`` command returns the details for every auth entry configured for rho. This output includes the name, username, password, ssh keyfile and sudo password for each entry. Passwords are masked if provided, if not they will appear as ‘null’.
+The ``rho auth list`` command returns the details for every authentication profile that is configured for Rho. This output includes the name, user name, password, SSH keyfile and sudo password for each entry. Passwords are masked if provided, if not, they will appear as ``null``.
 
 **rho auth list [--vault=** *vault_file* **]**
 
 ``--vault=vault_file``
 
-  This contains the path and filename of the file containing the vault password. If this option is used the file should be limited in access on the system.
+  Contains the path of the file that contains the vault password. Because the encrypted Rho data could contain sensitive information, make sure that this vault password file is stored in a location that has limited access.
 
-The ``rho auth show`` command is the same as the ``rho auth list`` command, except that it only returns details for a single specified auth entry.
+The ``rho auth show`` command is the same as the ``rho auth list`` command, except that it returns details for a single specified authentication profile.
 
 **rho auth show --name=** *name* **[--vault=** *vault_file* **]**
 
 ``--name=name``
 
-  This required argument gives the authentication credentials entry to display.
+  Required. Contains the authentication profile entry to display.
 
 ``--vault=vault_file``
 
-  This contains the path and filename of the file containing the vault password. If this option is used the file should be limited in access on the system.
+  Contains the path of the file that contains the vault password. Because the encrypted Rho data could contain sensitive information, make sure that this vault password file is stored in a location that has limited access.
 
-Deleting Authentication Credentials
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Clearing Authentication Profiles
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-It can be necessary to remove authentication credentials as the network infrastructure changes. This is done using the ``clear`` subcommand.
+As the network infrastructure changes, it might be necessary to delete some authentication profiles. Use the ``clear`` subcommand to delete authentication profiles.
 
-**IMPORTANT:** Remove the auth setting from any profile which uses it *before* removing the auth entry. Otherwise, any attempt to use the profile attempts to use the non-existent auth entry, which causes the ``rho`` command to fail.
+**IMPORTANT:** Remove or change the authentication profile from any network profile that uses it *before* clearing an authentication profile. Otherwise, any attempt to use the network profile to run a scan runs the command with a nonexistent authentication profile, an action that causes the ``rho`` command to fail.
```

Comment:
May want to cross reference what section tells you how to do that. (`rho profile edit ...`)